frame/balances/src/lib.rs

```diff
@@ -376,11 +357,9 @@ decl_storage! {
 		///
 		/// NOTE: THIS MAY NEVER BE IN EXISTENCE AND YET HAVE A `total().is_zero()`. If the total
 		/// is ever zero, then the entry *MUST* be removed.
-		pub Account get(fn account)
-			build(|config: &GenesisConfig<T, I>| config.balances.iter()
-				.map(|&(ref who, free)| (who.clone(), AccountData { free, .. Default::default() }))
-				.collect::<Vec<_>>()
-			): map hasher(blake2_256) T::AccountId => AccountData<T::Balance>;
+		///
+		/// NOTE: This is only used in the case that this module is used to store balances.
```

Comment:
So you could specify Balances as implementing the `AccountStore` trait in the pallet configuration, and account balances would be kept here and act like before?

Reply:
Am I missing something? Balances should implement StoredMap?

Reply:
It seems StorageAccountShim does the magic here per the "local test". If this is right, you can resolve this conversation.